uktrade/data-hub-frontend
# There's now only one journey throuth the review form

## Description of change

There's now only one journey throuth the review form i.e. if the user checks the _Some of this information needs revising_ radio option, they won't see a truncated version of the form, but pretty much the same form as if they selected the _I confirm this information is correct_ option, except that the _Comments_ textarea field is required.

## Test instructions

1. Go to `/exportwins/review/<token>`
2. Selec the _Some of this information needs revising_
3. Click the _Continue_ button
4. You should now be on _Step 2 of 6_ of the form and...
5. You should see a radio button group with the label _What value do you estimate you would have achieved without our support?_, and...
6. A textarea field with a _ Comments_ label, which should be required
7. Click the _Continue_ button
8. The rest of the form should be exactly the same as if you selected the _I confirm this information is correct_ option

_What should I see?_

## Screenshots

### Before

![single-journey-review-before](https://github.com/uktrade/data-hub-frontend/assets/2333157/9c2523e8-2ef0-4254-89a1-b5d229367d2f)

### After

![single-journey-review-after](https://github.com/uktrade/data-hub-frontend/assets/2333157/81c467ac-060e-43a9-b7b0-e7804f8a5e5b)
